src/app/components/internal/ComponentExample/ComponentExample.js

```diff
@@ -164,6 +182,34 @@ class ComponentExample extends Component {
 
     const componentLink = `https://codepen.io/team/carbon/full/${codepenSlug}/`;
     const counter = Math.floor(Math.random() * 100) + 1;
+    const reactLink = `http://react.carbondesignsystem.com/?selectedKind=${componentNameLink}`;
+    
+    
+
+    if (
+      component === 'checkbox' ||
+      component === 'label'
+    ) {
+      var angularLink = `http://angular.carbondesignsystem.com/?selectedKind=Forms&selectedStory=${componentNameLink}`;
```

Comment:
Great to have this tabular-ized, too, like:

```javascript
const angularLink = {
  ...
  notification: `http://angular.carbondesignsystem.com/?selectedKind=Banner`,
  toggle: `http://angular.carbondesignsystem.com/?selectedKind=Forms&selectedStory=switch`,
  ...
}[component];
```

Reply:
There is a default link so I wasn't able to use what you suggested here, but I did update the js to use switch. Seems a bit cleaner now?

Reply:
@alisonjoseph switch sounds OK - For the record, the “default” scenario can be covered by:

```javascript
{
  foo: fooValue,
  bar: barValue,
][key] || defaultValue;
```

Reply:
👍 thanks Akira! I didn't realize 😄